src/test/java/edu/ucsb/cs156/example/controllers/RecommendationControllerTests.java

```diff
@@ -168,7 +168,6 @@ public void an_admin_user_can_post_a_new_recommendation() throws Exception {
             LocalDateTime ldt1 = LocalDateTime.parse("2022-01-03T00:00:00");
 
             Recommendation one = Recommendation.builder()
-                            .id(1)
```

Comment:
There are several other places where the `.id` method is called on the Recommendation builder.  Do those all have to change?

Reply:
> There are several other places where the `.id` method is called on the Recommendation builder. Do those all have to change?

<img width="531" alt="image" src="https://user-images.githubusercontent.com/44068328/201288620-3fdb2891-7c2a-431f-9794-6a6397e09fb2.png">

Reply:
Just for POST